account/login/login.html

```diff
@@ -68,7 +68,7 @@ <h1>Login</h1>
 
     <div class="login-btns">
         <a href="#" class="login">Login</a>
-        <a href="/account/sign-up/signup.html" class="new-account">Create New Account <strong>></strong></a>
+        <a href="/account/sign up/signup.html" class="new-account">Create New Account <strong>></strong></a>
     </div>
```

Comment:
You've added a space in the name of folder, it's a good practice not to do so. Try avoiding using such naming conventions.

Reply:
Noted